src/main.ts

```diff
@@ -46,9 +46,9 @@ MainCommandOptions = {}) => {
         'remote',
         'set-url',
         'origin',
-        `${url.protocol}//${username}:${GITLAB_TOKEN!}@${
+        `${url.protocol}//${username}:${GITLAB_TOKEN}@${
           url.host
-        }${url.pathname.replace(/\/$/, '')}/${CI_PROJECT_PATH!}.git`,
+        }${url.pathname.replace(/\/$/, '')}/${CI_PROJECT_PATH}.git`,
```

Comment:
```suggestion
        }${url.pathname.replace(/\/$/, '')}/${env.CI_PROJECT_PATH}.git`,
```

It would be cleaner?

Reply:
Sure, I am OK with both.